### DISCUSSION TITLE
Grid stylish in my CMS

### DISCUSSION THREAD

**Discussion by emiliosh** [awcodes/filament-tiptap-editor#172] [Q&A] [answered]
Hi, I'm using your editor with Fabricator plugin for my CMS.

When I design a Grid in tiptap editor and then go to my CMS and render in blade, the style isn't applied.   What I'm doing wrong? I had the CSS and I guess I'm missing some tiptap js or whatever for rendering the style defined by filament-tiptap-editor plugin (something like filament-tiptap-grid__column).

Thanks in advance!

**Comment by awcodes** [comment]
It's up to you to style the content outside the editor. I can't assume anything about the styling needs of your html. 🙂

**Reply by emiliosh** [reply, marked as answer]
I found the problem.

```
<div 
   class="filament-tiptap-grid-builder" 
   data-type="responsive" 
   data-cols="2" 
   style="grid-template-columns: repeat(2, 1fr);" 
   data-stack-at="md">
       <div class="filament-tiptap-grid-builder__column" data-cols="1" style="grid-column: span 1;"><p>hola</p></div>
       <div class="filament-tiptap-grid-builder__column" data-cols="1" style="grid-column: span 1;"><p>adios</p></div>
</div>
```
In style at outer div tag the _display: grid_ is missing.  I had to create a _filament-tiptap-grid-builder_ class with _display:grid_ attribute.

Thanks a lot for your fast response.
